src/Monitoring/MetricsAgent.php

```diff
@@ -13,4 +13,12 @@
 interface MetricsAgent
 {
     public function addTimer(MetricId $metric, DateTimeImmutable $start_time, DateTimeImmutable $end_time): void;
+
+    public function addSample(MetricId $metric, float $value): void;
```

Comment:
Worth a quick docblock on each of these explaining the different types maybe?

Reply:
Yeah I've still to do a bit of a write up across the board and need to research again the types and explanations of what we support in statsite and don't to remind us why we called key-values counterIntegrals and why samples are mapped to timers